yorkie/src/main/kotlin/dev/yorkie/api/ChangeConverter.kt

```diff
@@ -0,0 +1,82 @@
+package dev.yorkie.api
+
+import com.google.protobuf.kotlin.toByteStringUtf8
+import dev.yorkie.document.change.Change
+import dev.yorkie.document.change.ChangeID
+import dev.yorkie.document.change.ChangePack
+import dev.yorkie.document.change.CheckPoint
+import dev.yorkie.document.time.ActorID
+
+typealias PBChange = dev.yorkie.api.v1.Change
+typealias PBChangeID = dev.yorkie.api.v1.ChangeID
+typealias PBCheckPoint = dev.yorkie.api.v1.Checkpoint
+typealias PBChangePack = dev.yorkie.api.v1.ChangePack
+
+internal fun List<PBChange>.toChanges(): List<Change> {
+    return map {
+        Change(it.id.toChangeID(), it.operationsList.toOperation(), it.message)
+    }
+}
+
+internal fun Change.toPBChange(): PBChange {
+    return PBChange.newBuilder().apply {
```

Comment:
we should take leverage of Dsl's provided by protubuf kotlin :)

for example we could change this to below;
```
change {
    id = this@toPBChange.id.toPBChangeID()
    message = this@toPBChange.message
    operations.addAll(this@toPBChange.operations.toPBOperations())
}
```